### PR TITLE
Hotfix: AWI-CM3 v3.1.3 fixing L1PCTCO2 

### PR DESCRIPTION
**Pull request by JanStreffing**
L1PCTCO2 switch was found to be not working by @Mccino.

Issue fixed. This can probably be merged right away.

### CODE REVIEW

File: runscripts/awicm3/v3.1.3/awicm3-v3.1.3-levante-TCO95L91-CORE2.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+general:
+    user: !ENV ${USER}
```
Comment (pgierz) [inline runscripts/awicm3/v3.1.3/awicm3-v3.1.3-levante-TCO95L91-CORE2.yaml:2]: I love that this is being used :-)

**Review by pgierz — APPROVED** [review]
All good, can't see any problems. The `requires` is something new for me, but maybe you and @mandresm built that while I was busy with other things.

**Comment by JanStreffing** [comment]
> All good, can't see any problems. The `requires` is something new for me, but maybe you and @mandresm built that while I was busy with other things.

I'm not even really sure that does anything. I think I put requires into the very first version of this yaml file together with Dirk still. If you have never seen it anywhere else, I'm not sure that it's required (no pun intended ;-))

**Comment by JanStreffing** [comment]
#bump

**Comment by JanStreffing** [comment]
!bump

**Comment by mandresm** [comment]
#approve-changes

**Pull request merged**